frontend/src/component/layout/MainLayout/AdminMenu/AdminMenuIcons.tsx

```diff
@@ -0,0 +1,43 @@
+import type SvgIcon from '@mui/material/SvgIcon/SvgIcon';
+import HomeIcon from '@mui/icons-material/Home';
+import LaptopIcon from '@mui/icons-material/Laptop';
+import EventNoteIcon from '@mui/icons-material/EventNote';
+import BillingIcon from '@mui/icons-material/CreditCardOutlined';
+import PeopleOutlineRoundedIcon from '@mui/icons-material/PeopleOutlineRounded';
+import KeyRoundedIcon from '@mui/icons-material/KeyRounded';
+import CloudIcon from '@mui/icons-material/Cloud';
+import HubOutlinedIcon from '@mui/icons-material/HubOutlined';
+import BuildOutlinedIcon from '@mui/icons-material/BuildOutlined';
+import EmptyIcon from '@mui/icons-material/CheckBoxOutlineBlankOutlined';
+import type { FC } from 'react';
+
+const icons: Record<string, typeof SvgIcon> = {
+    '/admin': HomeIcon,
+    users: PeopleOutlineRoundedIcon,
+    '/admin/service-accounts': LaptopIcon,
+    access: KeyRoundedIcon,
+    sso: CloudIcon,
+    network: HubOutlinedIcon,
+    instance: BuildOutlinedIcon,
+    '/admin/billing': BillingIcon,
+    '/history': EventNoteIcon,
+};
+
+const findIcon = (key: string) => {
+    return icons[key] || EmptyIcon;
+};
+
+export const IconRenderer: FC<{ path: string; active: boolean }> = ({
```

Comment:
Stolen with pride from NavigationSidebar

Reply:
Maybe it should be an `AdminIconRenderer`? Spare some time for devs who use auto-import by component name :P

Reply:
Yeah not a bad point, I'll have a look at that in a later PR